mail_tracking/static/src/models/messaging_initializer.esm.js

```diff
@@ -5,73 +5,12 @@ import {registerPatch} from "@mail/model/model_core";
 registerPatch({
```

Comment:
I think this commit could be squashed with the migration one, usually there's only one migration commit.